kaguya/assets/css/style.css

```
@@ -0,0 +1,349 @@
+@charset "utf-8";
+
+/* ベース設定 */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 1280px;
+  margin: 0 auto;
+}
+
+a {
+  color: #b2b2b2;
+  text-decoration: none;
+}
+
+img {
+  max-width: 100%;
+}
+
+/* ヘッダー */
+.ly_header {
+  height: 56px;
+}
+
+.ly_header_inner {
+  height: inherit;
+  padding: 16px 96px 8px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+.bl_headerNav {
+  display: flex;
+}
+
+.bl_headerNav_item {
+  padding-left: 56px;
+  list-style: none;
+}
+
+.bl_herderNav_link_checked {
+  color: #3b3b3b;
+}
+
+/* メイン背景 */
+
+.ly_mainBackGroundImage {
+  height: 744px;
+  background-image: url(../images/home_top.png);
+  position: relative;
+}
+
+.bl_mainBackGroundImage {
+  position: absolute;
+  top: 37%;
+  width: 609px;
+  height: 137px;
+  padding: 32px 32px 32px 88px;
+  background: #ffffff 0% 0% no-repeat padding-box;
+  opacity: 0.8;
+}
+
+.bl_mainBackGroundImage_title {
+  margin: 0;
+  color: #918f89;
+  font-weight: normal;
+  font-size: 53px;
+  line-height: 73px;
+  text-align: right;
+}
+
+/* メインメッセージ */
+.ly_mainMessage {
+  padding: 80px 0px 0px 88px;
+  margin-bottom: 240px;
+  display: flex;
+}
+
+.bl_mainMessage_content {
+  padding: 240px 0px 0px 56px;
+}
+
+.bl_mainMessage_content_title {
+  padding-bottom: 71px;
+  margin: 0;
+  color: #3b3b3b;
+  font-weight: normal;
+  font-size: 25px;
+  line-height: 53px;
+  letter-spacing: 2.5px;
+}
+
+.bl_mainMessage_content_sentence {
+  margin: 0;
+  color: #3b3b3b;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 30px;
+  letter-spacing: 0.36px;
+}
+
+/* ニュースコンテンツ */
+.ly_news {
+  padding: 0px 88px;
+  margin-bottom: 240px;
+}
+
+.bl_newsContent_title {
+  padding-bottom: 30px;
+  margin: 0;
+  color: #3b3b3b;
+  font-weight: normal;
+  font-size: 23px;
+  line-height: 53px;
+  letter-spacing: 2.3px;
+  text-align: center;
+}
+
+.bl_newsContent_list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.bl_newsContent_item {
+  list-style: none;
+  position: relative;
+}
+
+.bl_newsContent_item_wrapper {
+  position: absolute;
+  top: 0px;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #000000 0% 0% no-repeat padding-box;
+  opacity: 0.4;
+}
+
+.bl_newsContent_item_p {
+  color: #ffffff;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 50px;
+}
+
+.el_viewMore {
+  text-align: center;
+  padding: 40px 0px 0px;
+  color: #d9d9d9;
+  font-size: 20px;
+}
+
+.el_viewMore > a::after {
+  color: #dfdfdf;
+  content: " >";
```

結果として表示は変わりませんでしたが、contentにスペースを入れるのはやめておきましょう。
（スペースによる位置調整は難しいので、スペース分必要になるなら、padding等で調整します）

```suggestion
  content: ">";
```